test/test_pcaprub_unit.rb

```diff
@@ -122,7 +122,7 @@ def test_create_from_primitives
 
   def test_set_datalink
     d = Pcap.lookupdev
-    o = Pcap.open_live(d, 65535, true, -1)
+    o = Pcap.open_live(d, 65535, true, 1)
```

Comment:
Fixes the following issue on macOS:

```
E
===============================================================================
Error: test_set_datalink(PCAPRUB::Pcap::UnitTest): RuntimeError: en0: BIOCSRTIMEOUT: Unknow
rake aborted!
/Users/runner/work/pcaprub/pcaprub/test/test_pcaprub_unit.rb:125:in `open_live'
Command failed with status (1): [ruby -w -I"lib:lib:test" /Users/runner/hostedtoolcache/Ruby/3.0.2/x64/lib/ruby/gems/3.0.0/gems/rake-13.0.6/lib/rake/rake_test_loader.rb "test/test_pcaprub_unit.rb" ]
/Users/runner/work/pcaprub/pcaprub/test/test_pcaprub_unit.rb:125:in `test_set_datalink'
/Users/runner/hostedtoolcache/Ruby/3.0.2/x64/bin/bundle:23:in `load'
     122:
/Users/runner/hostedtoolcache/Ruby/3.0.2/x64/bin/bundle:23:in `<main>'
     123:   def test_set_datalink
Tasks: TOP => default => test
     124:     d = Pcap.lookupdev
(See full trace by running task with --trace)
  => 125:     o = Pcap.open_live(d, 65535, true, -1)
     126:     dls = o.listdatalinks
     127:     begin
     128:       assert_equal(o,o.setdatalink(dls.values.first))
===============================================================================
.
```

From a quick Google it looked like `-1` as a timeout was invalid; And all of the other `Pcap.open_live` calls are set to `1`, so I've updated this here too for consistency. The macOS tests pass after this change.